people.json

```diff
@@ -1124,5 +1124,47 @@
             "Android SDK",
             "Xamarin.Forms"
         ]
+    },
+
+    "Bryan Gillis": {
+        "interests": [
+            "Python (programming language)",
+            "LaTeX",
+            "Markdown",
+            "Git",
+            "GitHub",
+            "Astrophysics",
+            "Bash (Unix shell)",
+            "Windows Subsystem for Linux",
+            "Microsoft Windows",
+            "C Sharp (programming language)",
+            "Project management",
+            "JSON",
+            "Software design",
+            "Eclipse (software)",
+            "Java (programming language)",
+            "C (programming language)",
+            "C++",
+            "Continuous integration",
+            "Linux",
+            "VS Code",
+            "HTML",
+            "Cascading Style Sheets",
+            "JavaScript",
+            "Web application",
+            "Web development",
+            "HPC",
+            "HTC",
+            "VirtualBox",
+            "Microsoft Azure",
+            "Amazon Web Services",
+            "Cloud computing",
+	    "Unit Testing",
+	    "Integration Testing",
+	    "Profiling",
+	    "Optimisation",
+            "Verification and Validation",
+	    "Code Pipelines"
```

Comment:
The test doesn't find 'Code Pipelines'. Would [Software Pipelining](https://en.wikipedia.org/wiki/Software_pipelining) be equivalent for your purpose?

Reply:
It looks like the Wikipedia article covering what I meant is actually https://en.wikipedia.org/wiki/Pipeline_(computing) - changed it to this